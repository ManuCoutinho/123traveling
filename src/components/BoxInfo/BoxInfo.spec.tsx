import { screen, render } from '@testing-library/react'
import { BoxInfo } from '.'

const mockProps = {
	city: 'angels',
	country: 'neverland',
	language: '1234'
}
describe('<BoxInfo />', () => {
	it('should render BoxInfo correctly ', async () => {
		render(<BoxInfo {...mockProps} />)
		expect(screen.getByText(/angels/)).toBeInTheDocument()
		expect(screen.getByText(/neverland/)).toBeInTheDocument()
		expect(screen.getByText(/1234/)).toBeInTheDocument()
	})
	it('should match to snapshot', () => {
		const { container } = render(<BoxInfo {...mockProps} />)
		expect(container.firstChild).toMatchSnapshot()
	})
})
