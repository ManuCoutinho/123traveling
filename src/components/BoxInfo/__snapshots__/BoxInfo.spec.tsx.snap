// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoxInfo /> should match to snapshot 1`] = `
<div
  class="chakra-stack css-hxyc7v"
>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-1kjoj6d"
    >
      neverland
    </p>
    <div
      class="chakra-stack css-84zodg"
    >
      <p
        class="chakra-text css-rrte29"
      >
        pa√≠ses
      </p>
      <span />
    </div>
  </div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-1kjoj6d"
    >
      1234
    </p>
    <div
      class="chakra-stack css-84zodg"
    >
      <p
        class="chakra-text css-rrte29"
      >
        idiomas
      </p>
      <span>
        <span>
          <svg
            color="#47585B"
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            style="color: rgb(71, 88, 91);"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
  <div
    class="css-0"
  >
    <p
      class="chakra-text css-1kjoj6d"
    >
      angels
    </p>
    <div
      class="chakra-stack css-84zodg"
    >
      <p
        class="chakra-text css-rrte29"
      >
        cidades +100
      </p>
      <span>
        <span>
          <svg
            color="#47585B"
            fill="none"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            style="color: rgb(71, 88, 91);"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
</div>
`;
